nucypher/cli/commands/worklock.py

```diff
@@ -120,6 +121,12 @@ def bid(general_config, worklock_options, registry_options, force, hw_wallet, va
     """Place a bid, or increase an existing bid"""
     emitter = _setup_emitter(general_config)
     registry = registry_options.get_registry(emitter, general_config.debug)
+
+    worklock_agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)  # type: WorkLockAgent
+    now = maya.now().epoch
```

Comment:
Just want to point that in agent we get "now" as blockchain time and I don't really know what way is better: because if they are not equal that means not synced geth